### PR TITLE
Contributing updates

### PR DESCRIPTION
**Pull request by tas50**
There's still a lot to improve here, but this is closer to reality

### CODE REVIEW

**Review by rshade — APPROVED** [review]

File: content/contributing.md [hunk 1]
```diff
@@ -17,33 +17,33 @@ It is a best practice to have your commit message have a _summary line_, followe
...
+We don't require you to use GitHub, and we will even take patch diffs attached to tickets on the issue tracker. However GitHub has a lot of convenient features, such as being able to see a diff of changes between a pull request and the main repository quickly without downloading the branch.
 
 ## Functional and Unit Tests
 
-This cookbook is set up to run tests under [Test Kitchen](http://kitchen.ci/). It uses serverspec to run integration tests after the node has been converged to verify that the state of the node.
+Our cookbooks are set up to run tests under [Test Kitchen](http://kitchen.ci/). They use InSpec to run integration tests after the nodes have converged to verify that the system state..
 
 Test kitchen should run completely without exception using the default [baseboxes provided by Chef](http://chef.github.io/bento/). Because Test Kitchen creates VirtualBox machines and runs through every configuration in the .kitchen.yml file, it may take some time for these tests to complete.
```
Comment (shortdudey123) [inline content/contributing.md]: maybe add info about kitchen-dokken here was well? `.kitchen.dokken.yml`

**Review by shortdudey123 — APPROVED** [review]
LGTM other than the one thought

File: content/contributing.md (hunk 1 above)
Comment (tas50) [reply content/contributing.md]: I think the whole area really needs to get rewritten. The goal here was to nudge it closer to reality. It should mention our complete test strategy not just local test

**Comment by tas50** [comment]
@shortdudey123 @rshade I added some words

**Comment by shortdudey123** [comment]
👍

**Comment by rshade** [comment]
:shipit:

**Pull request merged**